ducc/webhook/registry-webhook.py

```diff
@@ -0,0 +1,111 @@
+from flask import Flask
+from flask import request
+
+import argparse
+import pprint
+import os
+
+app = Flask(__name__)
+
+@app.route("/<path:p>", methods=["POST"])
+def catch_all(p):
+    try:
+        for (action, image) in handle_harbor(request.json):
+            publish_message(notifications_file, action, image)
+
+        return "ok"
+    except Exception as e:
+        print("Fail to handle the harbor hook")
+        print(e)
+
+    try:
+        for (action, image) in handle_dockerhub(request.json, notifications_file):
+
+            publish_message(notifications_file, action, image)
+
+        return "ok"
+    except Exception as e:
+        print("Fail to handle the dockerhub hook")
+        print(e)
+
+    pprint.pprint(request.json)
+    return "ko", 500
+
+def publish_message(notifications_file, action, image):
+
+    with open(notifications_file, 'a+') as f:
+        if os.stat(notifications_file).st_size == 0:
+            current_id = 0
+        else:
+            f.seek(0)
+            last_line = f.readlines()[-1]
+            last_line_id = last_line.split('|')[0]
+            current_id = int(last_line_id) + 1
+            if (int(last_line_id) % 1000 == 0 and int(last_line_id) != 0):
```

Comment:
The `1000` should become a command line parameter. We need it small for testing, and large for production.